mkosi/__init__.py

```diff
@@ -2755,23 +2814,35 @@ def jj(line: str) -> str:
 
 
 def install_boot_loader_clear(args: CommandLineArguments, root: str, loopdev: str) -> None:
-    nspawn_params = [
-        # clr-boot-manager uses blkid in the device backing "/" to
-        # figure out uuid and related parameters.
-        f"--bind-ro={loopdev}",
-        f"--bind-ro=/dev/block",
-        f"--bind-ro=/dev/disk",
-        f"--property=DeviceAllow={loopdev}",
-    ]
-
-    for partno in (args.esp_partno, args.bios_partno, args.root_partno, args.xbootldr_partno):
-        if partno is not None:
-            p = partition(loopdev, partno)
-            nspawn_params += [f"--bind-ro={p}", f"--property=DeviceAllow={p}"]
+    # clr-boot-manager uses blkid in the device backing "/" to
+    # figure out uuid and related parameters.
+    nspawn_params = nspawn_params_for_blockdev_access(args, loopdev)
 
     cmdline = ["/usr/bin/clr-boot-manager", "update", "-i"]
     run_workspace_command(args, root, cmdline, nspawn_params=nspawn_params)
 
+def install_boot_loader_centos_old_efi(args: CommandLineArguments, root: str, loopdev: str) -> None:
+    nspawn_params = nspawn_params_for_blockdev_access(args, loopdev)
+
+    # prepare EFI directory on ESP
+    os.makedirs(os.path.join(root, 'efi/EFI/centos'), exist_ok=True)
```

Comment:
Do we need this directory specifically? What will grub use as the default?

Reply:
> Looks great! A few more comments

Mh, I'm unable to comment on the `mkosi.md` remarks. Indeed I seem to have forgotten to extend that, only reworded the commit message. Gonna fix that tomorrow.

Reply:
Seems you replied to the wrong comment here (or Github did something weird)